ice_setup/ice.py

```diff
@@ -567,6 +575,7 @@ def create_repo_file(cls, template_name, repo_url, gpg_url, file_name=None, **kw
             contents = template.format(
                 gpg_url=gpg_url,
                 repo_url=repo_url,
+                gpg_check=1 if use_gpg else 0,
```

Comment:
yeah, same philosophical question. When to put it in the method signature, or when to put it in kwargs. I like to see the args spelled out so that you can see them and know they are there easily.  Otherwise you have to look through the function to know what magic options you can also pass.

I know it's not magic -- you need to document the function properly.  If we did use kw.get() here, what about all the other ones that I changed?  I'm curious to find a best practice/recommendation for this.